data/mt/retired/Denise-Hayman-482e35af-a664-4010-a5db-33f21d902045.yml

```diff
@@ -3,22 +3,24 @@ name: Denise Hayman
 given_name: Denise
 family_name: Hayman
 gender: Female
-email: denise.hayman@mtleg.gov
+email: Denise.Hayman@legmt.gov
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-31'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '66'
-offices:
-- classification: capitol
-  address: PO BOX 6115;Bozeman, MT 59771-6115
-  voice: 406-579-1986
+- end_date: '2022-12-31'
+  type: upper
```

Comment:
@bfossen-ce this person was in the house from 2015 to 01/02/2023. On 01/02/2023 they will start in the Montana senate